pkg/apiserver/apiserver.go

```diff
@@ -100,10 +100,10 @@ func (cfg *Config) Complete() CompletedConfig {
 
 func (config completedConfig) New() (*ClusterPediaServer, error) {
 	if config.ClientConfig == nil {
-		return nil, errors.New("")
+		return nil, errors.New("ClientConfig is missing")
 	}
 	if config.StorageFactory == nil {
-		return nil, errors.New("")
+		return nil, errors.New("StorageFactory config is missing")
```

Comment:
We could use the same error style as `genericapiserver`, after all, they are functions with the same meaning.
https://github.com/clusterpedia-io/clusterpedia/blob/01ad71bfa087b18e16eb6c981c874c023c91acff/vendor/k8s.io/apiserver/pkg/server/config.go#L592-L601

eg. `apiserver.New() called with config.ClientConfig == nil`
> `Clusterpediaapiserver.New` and  `APIServer.New` don't look good, we can just use the `apiserver.New`

Reply:
I have revised it as suggested .